dll/shellext/zipfldr/zipfldr.spec

```diff
@@ -2,4 +2,4 @@
 @ stdcall -private DllGetClassObject(ptr ptr ptr)
 @ stdcall -private DllRegisterServer()
 @ stdcall -private DllUnregisterServer()
-@ stdcall RouteTheCall(ptr ptr wstr long)
\ No newline at end of file
+@ stdcall RouteTheCallW(ptr ptr wstr long)
```

Comment:
Doesn't the now-absence of `RouteTheCall` (non-W) break compatibility with Windows' own zipfldr extension?
The non-W version would work the ANSI string, so you could always have the new W version, and implement the ANSI version by converting the strings to unicode.

Reply:
> break compatibility with Windows' own zipfldr extension

Yes and no. Rundll32 on Windows NT has always tried to find the W version first. If somebody manually looks for RouteTheCall without W with GetProcAddress then yes but that makes no sense for anyone to do.

 I already suggested adding a non-W export but @JoachimHenze wanted me to wait.

Reply:
Ftr: The zipfldr of XPSP3 has only **one** of those exports. It is a non_W-version.
![image](https://github.com/reactos/reactos/assets/33393466/02ca2a2f-8a72-455f-8f3a-e981dd164818)

Reply:
> The zipfldr of XPSP3 has only one of those exports. It is a non_W-version.

And it does not support Unicode paths (that is, characters that cannot be represented in CP_ACP). At [this point in time](https://devblogs.microsoft.com/oldnewthing/20180515-00/?p=98755) zipfldr was a 3rd-party component provided by DavePl and I don't think Microsoft had any control over how it worked.

In later Windows versions, RouteTheCall is not used and [seems to just ShellExecute its input string](https://lolbas-project.github.io/lolbas/Libraries/Zipfldr/). On a .zip file that means DelegateExecute now in Win10.